### DISCUSSION TITLE
nova-2-ea periods of silence filled with nonsense

### DISCUSSION THREAD

**Discussion by peterjkirby** [deepgram/community#404] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

My team and I use DeepGram to transcribe our customers meeting recordings. We recently began to explore switching the model we were using from Nova to Nova 2. 

Once thing we noticed was that, during some periods of silence in the recordings, the resulting transcripts contain nonsense. Since periods of silence aren't unusual in a meeting (e.g. everyone taking a break, waiting for people to arrive, etc), many of the transcripts contain multiple sections of nonsense words. 

For example, in a recent meeting where a meeting recording was started immediately, but participants didn't arrive and begin talking for about 11 minutes, the first 11 minutes of the transcript were filled with the following text:

> you and and and and and and and and and and and and and and and and and and and and and and and and and and and and and and and and and okay uh-huh uh-huh uh-huh uh-huh uh-huh and and and and and and and and and and and and and and and and and and and uh-huh and and and and and and and and and and and and and and and and and and

The timestamps associated indicate that each word was detected 6-9 seconds prior to the previous word. 

In recordings where we are seeing this output in transcripts, the audio during these periods of silence is truly quiet (there is no background noise). 

Nonsense output like this was not present when using the Nova 1 model.

There is a similar ticket to this one (https://github.com/orgs/deepgram/discussions/388), but I wanted to open up a separate issue because we are seeing these "hallucinations" intermixed with valid transcribed conversation. I wasn't sure if this was all a symptom of the same problem or not, but feel free to close this issue if you would prefer to consolidate conversation in that issue thread.

**Comment by jkroll-deepgram** [comment]
Hi @peterjkirby, thank you for the additional feedback on Nova-2. This is indeed similar to the other ticket you linked, and it's an observed pattern that this Nova-2 early access model sometimes transcribes silence as these repeated hallucinations. We are actively working on solidifying the model and handling edge cases like this in advance of its general-availability release.

If you are able to share a request ID for this example, I'd like to investigate it further. Are you saying that the timestamps on the hallucinations are not in chronological order with the timestamps of the the actual spoken words?

**Comment by jpvajda** [comment, marked as answer]
@peterjkirby  👋  Nova-2 was made GA (generally available) Please try again using `nova-2` and let us know if you are still having issues. The [Docs ](https://developers.deepgram.com/docs/model#nova-2) here outline the different ways you call our Nova-2 models depending on your needs. 

Also #388  was answered if that is helpful to you. 


> Some models can hallucinate, we have seen this with, for example, audio files that contain true silence (as opposed to background noise). We have an algorithm in the works which solves this particular issue, I can test if it will solve your particular issue too (seems likely)!
